### PR TITLE
bento-selector hero example

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: extensions/amp-selector/1.0/example/index.html [hunk 1]
```diff
@@ -1,36 +1,63 @@
...
+    <script src="https://cdn.ampproject.org/bento.js"></script>
+    <script async src="https://cdn.ampproject.org/v0/bento-selector-1.0.js" ></script>
+    <link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-selector-1.0.css" />
+<style>
+    :root {
+      --space-1: .5rem;  /* 8px */
+      --space-4: 2rem;   /* 32px */
+    }
+    body {
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```
Comment (kvchari) [inline extensions/amp-selector/1.0/example/index.html:19]: Is there a reason for this complex of a font-family? Seems unnecessary for the example

Comment (sebastianbenz) [reply extensions/amp-selector/1.0/example/index.html:19]: That's just the system fonts. The alternative would be to load an extra font file.

File: extensions/amp-selector/1.0/example/index.html [hunk 2]
```diff
@@ -1,36 +1,63 @@
...
-    <bento-selector class="sample-selector">
-      <ul>
-        <li option="1">Option 1</li>
-        <li option="2">Option 2</li>
-        <li option="3">Option 3</li>
-        <li option="4">Option 4</li>
-      </ul>
+    <bento-selector role="tablist" keyboard-select-mode="focus">
+      <div id="tab1" role="tab" aria-controls="tabpanel1" option selected>Tab one</div>
+        <div id="tabpanel1" role="tabpanel" aria-labelledby="tab1">Tab one content... </div>
```
Comment (kvchari) [inline extensions/amp-selector/1.0/example/index.html:56]: indentation is off

Comment (sebastianbenz) [reply extensions/amp-selector/1.0/example/index.html]: done

**Review by kvchari — COMMENTED** [review]
It seems like we're using the <bento-selector> component as a [tabbed-navigation](https://getbootstrap.com/docs/4.0/components/navs/#tabs), which is a cool use of it. But, from our existing [amp-selector](https://amp.dev/documentation/examples/components/amp-selector/?format=websites), I think the intended use of this component is more as an alternative to `<select>`. I like demonstrating interesting uses like this, but not sure if it makes sense as a the first example the consumers sees?

Couple of notes if we want to stick with this more complex example:
1. Serving this with `amp` doesn't seem to work - clicking on Tabs 2 and 3, doesn't show the tab 2/3 content
2. ~If we want to demonstrate tabbed ui with all the relevant `aria-` controls for accessibility, we might want to leverage [`aria-live` regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)~ Looks like it's not necessary https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel
3. Since we're changing the example in non-trivial ways, we might want to make the same changes to the README.md as well (or at least add this as a 2nd example)

**Comment by sebastianbenz** [comment]
I was wondering the same (I copied the sample from our amp examples). I like the example because it demonstrates the flexibility of amp-selector, but agree that it maybe is the wrong example to demo the basic component. What do you think would be an idiomatic example for bento-selector?

**Comment by kvchari** [comment]
Hmm not sure which amp samples you were looking at, but the examples of "classic" amp-selector [here](https://github.com/ampproject/amphtml/blob/main/examples/selector.amp.html) seem much simpler. Can you copy over any changes you make to the readme as well?

**Pull request merged**